lib/dynamodb/save.js

```diff
@@ -0,0 +1,16 @@
+import normalize from './normalize';
```

Comment:
Why a separate lib and not part of db service?

Reply:
"Db service" = DynamoDB table setup, no code is deployed (no lambdas). That's why I thought it would make more sense to decouple the library (dynamodb helpers) from the service